src/tasks/start/frontend/aragon-client.ts

```diff
@@ -15,29 +16,79 @@ export async function installAragonClientIfNeeded(
   repo: string = defaultRepo,
   version: string = defaultVersion
 ): Promise<string> {
-  // Determine client path.
   const clientPath: string = _getClientPath(version)
 
-  // Verify installation or install if needed.
-  if (fs.existsSync(path.resolve(clientPath))) {
-    logFront('Using cached client version.')
-  } else {
+  if (await _checkClientInstallationNeeded(clientPath)) {
```

Comment:
Changes here deal with failures during client installation, and attempt to detect malformed client installations. Users in the CLI were rarely being blocked by a client installation that failed, forcing them to manually remove directories in ~/.aragon. This is an attempt to try to avoid that.